app/models/goldencobra/widget.rb

```diff
@@ -188,6 +188,7 @@ def duplicate!
       attrs["title"] = "#{attrs["title"]} (Kopie)"
       attrs["id_name"] = attrs["id_name"].present? ? "#{attrs["id_name"]}-kopie" : nil
       attrs["active"] = false
+      attrs["tag_list"] = self.tag_list
```

Comment:
Would be nice if you can fix the one issue reported by CodeClimate:
<img width="835" alt="bildschirmfoto 2017-10-06 um 11 34 36" src="https://user-images.githubusercontent.com/1942953/31272181-59e90f8a-aa8a-11e7-902d-803a42e64ad6.png">